src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -7,7 +7,11 @@
           novalidate>
 
       <div i18n="form title"
-           class="form-header">{{ action | titlecase }} {{ resource | upperFirst }}</div>
+           class="card-header">{{ action | titlecase }} {{ resource | upperFirst }} 
+           <cd-help-text>
+           <span *ngIf="action === 'Copy'">Copy the content of a source image into the newly created destination image</span> 
+          </cd-help-text>
+          </div>
```

Comment:
these are in different component

Reply:
@prgoel-code then its a good reason to have it generalized in a file where it could be benefited by the entire rbd component. Maybe place it in the rbd.service file in the UI and get it from there? or even create a separate rbd.constants file which has a class to hold all these messages and later use it in these components. The reason is that whenever someone wants to change this in the future later on they'll have to look through different files and change it which is not good. Similar things should be generalized and reused everytime

Reply:
yes I was thinking to do this before.. is there any constant file already exist or we are using. there are multiple places where we are using messaged locally in component

Reply:
we don't have a constants for rbd but we have this [constants folder](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/shared/constants). You can probably create a new rbd constant file there and move it.